packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -39,10 +40,9 @@ const propTypes = {
    */
   rows: PropTypes.arrayOf(WorklistDataGridPropTypes.rowShape),
   /**
-   * String that specifies the default width for columns in the grid. Any valid CSS width value is accepted.
-   * To override the default value, provide width for the column that needs to be overridden.
+   * Number indicating the default column width in px. This value will be used if no overriding width value is provided on a per-column basis.
```

Comment:
Do we need to specify that it's an integer?

Reply:
No.